ts-sdk/whirlpool/src/createPool.ts

```diff
@@ -0,0 +1,276 @@
+ * 
+ * const devnetRpc = createSolanaRpc(devnet('https://api.devnet.solana.com'));
+ * const wallet = await generateKeyPairSigner();
+ * await devnetRpc.requestAirdrop(wallet.address, lamports(1000000000n)).send();
+ * 
+ * const tokenMintOne = "TOKEN_MINT_ADDRESS_1"; 
+ * const tokenMintTwo = "TOKEN_MINT_ADDRESS_2"; 
+ * const initialPrice = 0.01;
+ * 
+ * const { poolAddress, instructions, initializationCost } = await createSplashPoolInstructions(
+ *   devnetRpc,
+ *   tokenMintOne,
+ *   tokenMintTwo,
+ *   initialPrice,
+ *   wallet
+ * );
+ */
+export function createSplashPoolInstructions(
+  rpc: Rpc<GetMultipleAccountsApi & GetMinimumBalanceForRentExemptionApi>,
+  tokenMintOne: Address,
+  tokenMintTwo: Address,
+  initialPrice: number = 1,
+  funder: TransactionPartialSigner = DEFAULT_FUNDER,
+): Promise<CreatePoolInstructions> {
+  return createConcentratedLiquidityPoolInstructions(
+    rpc,
+    tokenMintOne,
+    tokenMintTwo,
+    SPLASH_POOL_TICK_SPACING,
+    initialPrice,
+    funder,
+  );
+}
+
+/**
+ * Creates the necessary instructions to initialize a Concentrated Liquidity Pool (CLMM) on Orca Whirlpools.
+ *
+ * @param {SolanaRpc} rpc - A Solana RPC client for communicating with the blockchain.
+ * @param {Address} tokenMintOne - The first token mint address to include in the pool.
+ * @param {Address} tokenMintTwo - The second token mint address to include in the pool.
+ * @param {number} tickSpacing - The spacing between price ticks for the pool.
+ * @param {number} [initialPrice=1] - The initial price of token 1 in terms of token 2.
+ * @param {TransactionPartialSigner} [funder=DEFAULT_FUNDER] - The account that will fund the initialization process.
+ * 
+ * @returns {Promise<CreatePoolInstructions>} A promise that resolves to an object containing the pool creation instructions, the estimated initialization cost, and the pool address.
+ *
+ * @example
+ * import { createConcentratedLiquidityPool } from '@orca-so/whirlpools';
+ * import { generateKeyPairSigner, createSolanaRpc, devnet } from '@solana/web3.js';
+ * 
+ * const devnetRpc = createSolanaRpc(devnet('https://api.devnet.solana.com'));
+ * const wallet = await generateKeyPairSigner();
+ * await devnetRpc.requestAirdrop(wallet.address, lamports(1000000000n)).send();
+ * 
+ * const tokenMintOne = "TOKEN_MINT_ADDRESS_1";
+ * const tokenMintTwo = "TOKEN_MINT_ADDRESS_2"; 
+ * const tickSpacing = 64;
+ * const initialPrice = 0.01;
+ * 
+ * const { poolAddress, instructions, initializationCost } = await createConcentratedLiquidityPool(
+ *   devnetRpc,
+ *   tokenMintOne,
+ *   tokenMintTwo,
+ *   tickSpacing,
+ *   initialPrice,
+ *   wallet
+ * );
+ */
+export async function createConcentratedLiquidityPoolInstructions(
+  rpc: Rpc<GetMultipleAccountsApi & GetMinimumBalanceForRentExemptionApi>,
+  tokenMintOne: Address,
+  tokenMintTwo: Address,
+  tickSpacing: number,
+  initialPrice: number = 1,
```

Comment:
related to : https://github.com/orca-so/whirlpools/pull/262#discussion_r1797537694 (f64 price)

yeah, I agree that there is no large difference between 100.00000 and 100.00001.